src/main/java/bookstore/controller/BookController.java

```diff
@@ -20,6 +23,7 @@
 import org.springframework.web.bind.annotation.ResponseStatus;
 import org.springframework.web.bind.annotation.RestController;
 
+@Tag(name = "Book management", description = "Endpoints for managing books")
 @RequiredArgsConstructor
 @RestController
 @RequestMapping(value = "/api/books")
```

Comment:
```suggestion
@RequestMapping("/books")
```

Add server.servlet.context-path=/api in application.properties